crds/jwst/specs/fgs_pars-detector1pipeline.rmap

```diff
@@ -0,0 +1,24 @@
+header = {
+    'classes' : ('Match', 'UseAfter'),
+    'derived_from' : 'Hand-made, SCSB, 202102',
+    'file_ext' : '.asdf',
+    'filekind' : 'PARS-DETECTOR1PIPELINE',
+    'filetype' : 'PARS-DETECTOR1PIPELINE',
+    'instrument' : 'FGS',
+    'mapping' : 'REFERENCE',
+    'name' : '',
+    'observatory' : 'JWST',
+    'parkey' : (('META.VISIT.TSOVISIT',), ('META.OBSERVATION.DATE', 'META.OBSERVATION.TIME')),
+    'sha1sum' : '3993aa707809a2245ca5c15ce56a020bedfe4907',
+    'suffix' : 'pars-detector1pipeline',
+    'text_descr' : 'Step parameters for Detector1Pipeline',
+}
+
+comment = """
+Step parameter references are JWST step and pipeline configurations.
+This reftype provides the configuration for jwst.pipeline.Detector1Pipeline
+"""
+
+selector = Match({
+    'N/A' : 'N/A',
```

Comment:
What's the distinction between specs with a `'N/A' : 'N/A',` rule and those with an empty selector?  The default rule is new to me.

Reply:
Actually nothing and would like an opinion on this. Usually the initial rmaps are these spec rmaps. Because I did want this rule in the initial rmaps, because if there is not match for any pars-xxx reference, it should not fail but return N/A, I placed it in the specs only to not forget it.

So, it is really just a "remember to do this". It appears to have no effect, but not really sure.

Reply:
Hmm... looks like the selector doesn't make it into combined_specs.json.  Since that's the case, I don't think it can have an effect, because the software doesn't load up the individual files.

I'm slightly in favor of the empty selector but only because it matches our existing spec files and might save a bit of "what is this doing?" confusion in the future.  But I won't protest either way.